pkg/cloud/aggregation.go

```diff
@@ -8,39 +8,64 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-var aggregationVarNames = []string{
-	"K6_CLOUD_AGGREGATION_MIN_SAMPLES",
-	"K6_CLOUD_AGGREGATION_PERIOD",
-	"K6_CLOUD_AGGREGATION_WAIT_PERIOD",
-	"K6_CLOUD_METRIC_PUSH_INTERVAL",
-	"K6_CLOUD_MAX_METRIC_SAMPLES_PER_PACKAGE",
-	"K6_CLOUD_MAX_METRIC_PUSH_CONCURRENCY",
+var aggregationVarNames = map[int][]string{
+	1: []string{
+		// cloud output v1: to be removed in the future
+		"K6_CLOUD_AGGREGATION_MIN_SAMPLES",
+		"K6_CLOUD_AGGREGATION_PERIOD",
+		"K6_CLOUD_AGGREGATION_WAIT_PERIOD",
+		"K6_CLOUD_METRIC_PUSH_INTERVAL",
+		"K6_CLOUD_MAX_METRIC_SAMPLES_PER_PACKAGE",
+		"K6_CLOUD_MAX_METRIC_PUSH_CONCURRENCY",
+	},
+	2: []string{
+		// cloud output v2
+		"K6_CLOUD_API_VERSION",
+		"K6_CLOUD_AGGREGATION_PERIOD",
+		"K6_CLOUD_AGGREGATION_WAIT_PERIOD",
+		"K6_CLOUD_METRIC_PUSH_INTERVAL",
+		"K6_CLOUD_MAX_METRIC_PUSH_CONCURRENCY",
```

Comment:
```suggestion
		"K6_CLOUD_METRIC_PUSH_CONCURRENCY",
```
I guess you mixed some vars here

Reply:
OK, will fix that!